### PR TITLE
Delete unused protected parameters ndim, ndim2 and ndim_pointGravity

### LINKED ISSUE #4246: Unused ndim in MBS World
Does anybody knows what are the following protected parameters in `Modelica.Mechanics.MultiBody.World` for?
- ndim
- ndim2
- ndim_pointGravity

They are not used anywhere. Just being curious before deleting them.

@MartinOtter ?

### PR DESCRIPTION
**Pull request by tobolar**
Close #4246 

IMO no conversion script is needed for protected parameters.

### CODE REVIEW

**Comment by tobolar** [comment]
> If you like you can add it to the list of changed components (breakng compatibilty) already now, see https://github.com/modelica/ModelicaStandardLibrary/issues/4260.

Shall I better rebase this PR to [beutlich:create_Version_4_1_0](https://github.com/beutlich/ModelicaStandardLibrary/tree/create_Version_4_1_0)? It could make it easier to merge `package.mo` later on.

**Review by HansOlsson — APPROVED** [review]
Looks good.
I agree that an unused protected parameter can be removed, even if someone in theory could have used it.

**Comment by beutlich** [comment]
> > If you like you can add it to the list of changed components (breakng compatibilty) already now, see #4260.
> 
> @beutlich Shall I better rebase this PR to [beutlich:create_Version_4_1_0](https://github.com/beutlich/ModelicaStandardLibrary/tree/create_Version_4_1_0)? It could make it easier to merge `package.mo` later on.

No, just do it here.

**Comment by Harisankar-Allimangalath** [comment]
@beutlich can you please review the changes done by @tobolar , the milestone part is missing here and I hope its ok to go with MSL4.1.0  .  Thankyou

File: Modelica/package.mo [hunk 1]
```diff
@@ -2530,6 +2530,10 @@ The following <font color=\"blue\"><strong>existing components</strong></font> h
 </p>
 
 <table border=\"1\" cellspacing=\"0\" cellpadding=\"2\" style=\"border-collapse:collapse;\">
+<tr><td colspan=\"2\"><strong>Mechanics.MultiBody</strong></td></tr>
+<tr><td>World</td>
+    <td>Delete unused protected parameters <code>ndim</code>, <code>ndim2</code> and
+        <code>ndim_pointGravity</code>.</td></tr>
```
Comment (beutlich) [inline Modelica/package.mo:2536]:
```suggestion
    <td>The protected parameters <code>ndim</code>, <code>ndim2</code> and
        <code>ndim_pointGravity</code> have been removed.</td></tr>
```

**Review by beutlich — APPROVED** [review]
Fine for me (if release notes are slighty reformulated).

**Comment by Harisankar-Allimangalath** [comment]
@MartinOtter and @tobolar can this PR have the milestone addressing to MSl4.1.0.

**Comment by Harisankar-Allimangalath** [comment]
> Fine for me (if release notes are slighty reformulated).

@beutlich Are you referring to the auto generation of release notes? The implication you refer is not clear to us, so could you please clarify?

**Review by MartinOtter — APPROVED** [review]

**Comment by beutlich** [comment]
> > Fine for me (if release notes are slighty reformulated).
> 
> @beutlich Are you referring to the auto generation of release notes? The implication you refer is not clear to us, so could you please clarify?

No, I only meant https://github.com/modelica/ModelicaStandardLibrary/pull/4266/commits/0368d773597401440eb834ed4ae0a3d03042a420

**Pull request merged**